src/__tests__/category/category.mutation.test.js

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const categoryService = require('../../modules/category/category.service')
+const categoryResolver = require('../../modules/category/category.resolver')
+const client = require('../../utils/apolloClient');
+
+require('dotenv').config();
+
+let categoryId
+
+describe('mutations', () => {
+    test('should create new category', async () => {
+        const res = await client.mutate({ 
```

Comment:
When you get a response, please, check specific things
- data and all of its fields
- an array of errors (is it empty or does it contain any messages if you have provided invalid requests)
- HTTP status code if it is not already done by `apollo-client`